arielperez-app/src/main/java/edu/arielperez/advancedjava/model/StockQuote.java

```diff
@@ -40,30 +43,13 @@ public String getStockSymbol() {
 		return stockSymbol;
 	}
 
-	/**
-	 * 
-	 * @param stockSymbol the stockSymbol to set (a unique series of letters
-	 *                    representing the stock for trading purposes)
-	 */
-	public void setStockSymbol(String stockSymbol) {
-		this.stockSymbol = stockSymbol;
-	}
-
 	/**
 	 * @return the stockPrice (returns the price of the stock as of dateRecorded)
 	 */
 	public BigDecimal getStockPrice() {
 		return stockPrice;
 	}
 
```

Comment:
why the diffs here?

Reply:
trying to make the class immutable. no setters, only getters. variables are private and final. Once the object is instantiated it can't change.

Reply:
...guess I forgot to do it in the previous homework...